maquetado/setup/scss/layout.scss

```diff
@@ -1,15 +1,36 @@
+@import 'navbar';
+@import 'footer';
+
 .content {
+  grid-area: content;
   margin-left: 80px;
-  margin-top: 150px;
-  margin-bottom: 100px;
+}
+
+.site {
+  display: grid;
+  grid-row-gap: 50px;
+  grid-template-areas:
+    "navbar"
+    "content"
+    "footer";
+  grid-template-rows: $navbar-height auto $footer-height;
 }
 
 .tech-area {
   margin-top: 50px;
 }
 
 .tech-container {
-  display: flex;
-  flex-flow: row wrap;
-  margin-left: 70px;
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, $tech-container-width);
+  grid-template-rows: repeat(2, $tech-container-height);
+  margin-left: 200px;
+}
+
+.tech-container div:first-child {
```

Comment:
Wow, you are the first person that can do something like this w/o asking and a tip. Good job! 

Just do this inside `tech-container` as `&:first-child`